xgi/readwrite/bipartite.py

```diff
@@ -117,11 +116,12 @@ def read_bipartite_edgelist(
             delimiter=delimiter,
             create_using=create_using,
             nodetype=nodetype,
+            dual=dual,
         )
 
 
 def parse_bipartite_edgelist(
-    lines, comments="#", delimiter=None, create_using=None, nodetype=None
+    lines, comments="#", delimiter=None, create_using=None, nodetype=None, dual=False
 ):
     """
     A helper function to read a iterable of strings containing a bipartite edge list and
```

Comment:
Add description for the parameter `create_using` in docstring.

Reply:
done